x-pack/legacy/plugins/apm/public/components/shared/charts/CustomPlot/plotUtils.js

```diff
@@ -56,14 +56,17 @@ export function getPlotValues(
     return null;
   }
 
-  const xMin = d3.min(flattenedCoordinates, d => d.x);
-  const xMax = d3.max(flattenedCoordinates, d => d.x);
+  const xMin = start ? start : d3.min(flattenedCoordinates, d => d.x);
+
+  const xMax = end ? end : d3.max(flattenedCoordinates, d => d.x);
```

Comment:
Shouldn't `start` be called `xMin` and `end` be called `xMax`? And why the line breaks?